src/apps/vpn/ui/screens/devices/VPNDeviceListItem.qml

```diff
@@ -74,18 +74,21 @@ MZSwipeDelegate {
                 spacing: 0
 
                 MZInterLabel {
+                    Layout.fillWidth: true
+
                     text: name
+                    maximumLineCount: 1
                     elide: Text.ElideRight
```

Comment:
Strange that this wasn't enough to just work. I guess it needed the `.fillWidth`?

Reply:
referring to the `Text.ElideRight` that was already in place

Reply:
Yep, `Layout.fillWidth` is needed here. It is worth noting that setting `maximumLineCount: 1` or setting `wrapMode: Text.NoWrap` have the same effect and will both cause the text to elide